utils/ryzenai/notification_service.py

```diff
@@ -27,7 +27,7 @@
 
 sys.path.append(os.path.join(os.getcwd()))
 
-import tests.ryzenai.testing_utils as tu  # noqa
+import tests.ryzenai.testing_models as tu  # noqa
```

Comment:
`tu` is not a good name, you can import the required variables directly

Reply:
updated